wikimedia/wikipedia-ios
# Match edit summary content language to article language

**Phabricator:** https://phabricator.wikimedia.org/T324942

### Notes
* This PR changes the EditSummaryViewController and EditSaveViewController classes to match their content language to the edited article language

### Test Steps
1. Go to an article in any language other than the app's primary language
2. Edit the article to reach the summary view
3. Make sure the content of the view matches the article's language
